src/input-login.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { Text, TextInput, StyleSheet } from "react-native";
+
+interface InputProps {
+	inputHeader: string;
+	handleTextChange: Function;
+	value: string;
+	secureTextEntry?: boolean;
+}
+
+function InputLogin (props: InputProps): JSX.Element {
+  return (
+    <React.Fragment>
+			<Text>
+				{props.inputHeader}
+			</Text>
+    	<TextInput
+    	  style={styles.input}
+    	  value={props.value}
+				onChangeText={(value) => props.handleTextChange(value)}
+				secureTextEntry={props.secureTextEntry}
+    	/>
+    </React.Fragment>
+  );
+}
+
+const styles = StyleSheet.create({
+	input: {
+		height: 40,
+		marginBottom: 10,
+		padding: 10,
+		borderWidth: 1,
+	},
+});
+
+export default InputLogin;
```

Comment:
2 pontos aqui:
- faltou uma linha vazia no final pra remover os avisos do github
- a gente não costuma utilizar export default, default permite fazer o import com qualquer nome e isso pode gerar conflitos na hora de revisar/dar manutenção no código, então nesse caso pode fazer o export direto do componente

```suggestion
export function ...
```

Reply:
No caso, optei por colocar o export junto da declaração da função.